public/static/cookies.js

```diff
@@ -0,0 +1,17 @@
+// This function is executed when any page on the website finsihes loading and
```

Comment:
```suggestion
// This function is executed when any page on the website finishes loading and
```

Reply:
Thanks for the suggestion, but I think we can add it as a new issue with a good-first-issue label, this allows new contributors to get involved in the project and enable them to add value to the project. What do you say??

Reply:
Yeah good idea.